test/Metadata.cpp

```diff
@@ -50,7 +50,7 @@ string bytecodeSansMetadata(string const& _bytecode)
 bool isValidMetadata(string const& _metadata)
 {
 	Json::Value metadata;
-	if (!Json::Reader().parse(_metadata, metadata, false))
+	if (!dev::jsonParseStrict(_metadata, &metadata))
```

Comment:
This file is within the `dev` namespace.

Reply:
;)